invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html

```diff
@@ -6,50 +6,65 @@
   Invenio is free software; you can redistribute it and/or modify it
   under the terms of the MIT License; see LICENSE file for more details.
 #}
-{%- extends config.ACCOUNTS_COVER_TEMPLATE %}
+{%- extends "invenio_theme/page_cover.html" %}
 {% from "invenio_accounts/_macros.html" import render_field, form_errors %}
 
 {% block page_body %}
-
-  <div class="ui padded segments big form">
-    <div class="ui segment padded relaxed">
-      <div class="divider hidden"></div>
-
+<div class="ui centered stackable grid">
+  <div class="centered row">
+  <h2>{{ _('Create an Account') }}</h2>
+  </div>
+  <div class="ui divider"></div>
+  <div class="ui bottom attached segment sign-form">
+    <div class="ui two column stackable grid">
+      <div class="column">
+        <h4 class="ui inverted header">Citeable. Discoverable.</h4>
```

Comment:
Aren't these strings that user sees in the UI?
```suggestion
        <h4 class="ui inverted header">{{ _( Citeable. Discoverable.) }}</h4>
```

Reply:
This is the dummy text right now, which can be changed later,